### DISCUSSION TITLE
Running the model in home assistant

### DISCUSSION THREAD

**Discussion by caol-ila** [jomjol/AI-on-the-edge-device#1457] [Q&A]
Hi, 

I want to use the live stream of an IP camera for detecting the value of my water meter. I could run the model on "better" hardware inside my home assistant installation. Where can I find the ML model and how can I reuse it to run in Home Assistant? 

Thanks for your support!
Thomas

**Comment by jomjol** [comment]
That sounds like a hard topic. You can find the training models partially also on the GitHub (Programmed with TensorFlow):
https://github.com/jomjol/neural-network-digital-counter-readout
https://github.com/jomjol/neural-network-analog-needle-readout

The hybrid model (dig-cont...) is not public.
